src/components/MainSection/MainSection.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { MdLocationOn } from 'react-icons/md';
+
+import { useWeatherInfo } from 'WeatherInfoProvider';
+import WeatherIcon from 'components/WeatherIcon/WeatherIcon';
+import * as utilsFunc from 'utils/convertFunc'
+
+import styles from './MainSection.module.css';
+
+function MainSection(props) {
+  const weather = useWeatherInfo()
+  return (
+    <div className={styles.wrapper}>
+      <div className={styles.weatherIconWrapper}>
+        <WeatherIcon iconName={weather.currentWeatherForecast.weather[0].icon} />
+      </div>
+
+      <div className={styles.weatherInfoWrapper}>
+        <span className={styles.temperatureInfo}>
+          {utilsFunc.convertKevinToCelcius(weather.currentWeatherForecast.main.temp)}°C
```

Comment:
you should move the function call out of jsx (if the function call is a bit long) to improve readability